packages/core/src/styles/components/_tooltip.scss

```diff
@@ -0,0 +1,140 @@
+@import "~@carbon/motion/scss/motion.scss";
+@import "~@carbon/themes/scss/themes";
+@import '~@carbon/type/scss/type';
+@import '~@carbon/type/scss/font-face/mono';
+@import '~@carbon/type/scss/font-face/sans';
+
+div.cc-tooltip {
```

Comment:
Is this potentially too specific?

Reply:
suggesting that we remove the `div`  and keep the `.cc-tooltip` ?

Reply:
Yeah, and (again fairly generally) do we want to have `#{$prefix}--cc--tooltip` or similar?